pkg/common/config/option.go

```diff
@@ -68,6 +69,8 @@ type Options struct {
 	TraceLevel                   interface{}
 	ListenConfig                 *net.ListenConfig
 
+	TransporterNewer func(opt *Options) network.Transporter
```

Comment:
What does this name mean?

Reply:
function interface of 'creating a transporter'